crates/red_knot_workspace/Cargo.toml

```diff
@@ -15,7 +15,7 @@ license.workspace = true
 red_knot_python_semantic = { workspace = true }
 
 ruff_cache = { workspace = true }
-ruff_db = { workspace = true, features = ["os", "cache"] }
+ruff_db = { workspace = true, features = ["os", "cache", "testing"] }
```

Comment:
Oh, I think this is no longer necessary after https://github.com/astral-sh/ruff/pull/12985